stack/README.md

```diff
@@ -33,6 +33,28 @@ decisions made for other layers.
 * Use Gradle with Android Studio for building.
 * Use MVVM to model views.
 
+### iOS (Native)
+
+* Use Swift for writing native app code.
+* Use Xcode and xcodebuild for building iOS apps.
+* Use xcpretty to format xcodebuild output.
+* Use Xcode automatic provisioning when possible.
+* Use UIKit with Storyboards and MVVM for creating UIs.
```

Comment:
I've been trying to brush up on my iOS development and ran through some SwiftUI stuff and thought the experience was pleasant - I'm not a dedicated mobile developer but I'd endorse using SwiftUI in the future

Reply:
My understanding is SwiftUI is new enough that it's still an "experiment" for us.